invenio/modules/documents/utils.py

```diff
@@ -17,22 +17,19 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-"""Multimedia utilities."""
+"""Document utils."""
 
-import redis
 
-
-def initialize_redis():
-    """Initialize redis service.
-
-    .. note::
-
-        FIXME: Should be remove it and replaced with an invenio redis object.
-
-    """
-    from flask import current_app
-
-    _redis_server = redis.Redis.from_url(
-        current_app.config.get('CACHE_REDIS_URL', 'redis://localhost:6379')
+def uuid_to_path(uuid):
+    """Convert `~invenio.modules.documents.api.Document` to path."""
+    from invenio.modules.documents.api import Document
+    from invenio.modules.documents.errors import (
+        DocumentNotFound, DeletedDocument
     )
-    return _redis_server
+
+    try:
+        document = Document.get_document(uuid)
+    except (DocumentNotFound, DeletedDocument):
+        raise
+    else:
+        return document.get('uri', '')
```

Comment:
I'm not sure if the implementation can be so simple.

1. We need access rights that might depend on the record if the document is attached to any. (cc @lnielsen @tiborsimko)
2. The file can be on different storage e.g. S3, EOS, ... hence we should probably return something like `document.open('r')`. If we pass bytestream then we should amend Flask-IIIF so it can handle it too.